### PR TITLE
Fix/Reword autorenew cancellation

### PR DESCRIPTION
**Pull request by FredericEspiau**
## Reword autorew cancellation

Interal reference : SCFRCA-1237

### Description of the Change

Reworded some sentences and fix issue with billing frequencies not being displayed correcly

### CODE REVIEW

File: client/app/app.js [hunk 1]
```diff
@@ -109,7 +109,7 @@ angular
 
         $translateProvider.useMissingTranslationHandler("translateMissingTranslationHandler");
         $translateProvider.useLoaderCache(true);
-        $translateProvider.useSanitizeValueStrategy("sanitizeParameters");
+        $translateProvider.useSanitizeValueStrategy("escapeParameters");
```
Comment (FredericEspiau) [inline client/app/app.js]: Fixes an issue with UTF8 chars encoding while keeping security on: https://angular-translate.github.io/docs/#/guide/19_security

File: client/app/account/billing/common/renew-helper.service.js [hunk 2]
```diff
@@ -1,65 +1,74 @@
...
-
-    this.getRenewLabel = (service) => {
-        if (service === "0") {
-            return $translate.instant("autorenew_service_renew_0");
-        }
+        getRenewDateFormated (service) {
+            if (!service) {
+                return "";
+            } else if (this.serviceHasAutomaticRenew(service)) {
+                return _.capitalize(this.$filter("date")(service.expiration, "MMMM yyyy"));
```
Comment (jleveugle) [inline client/app/account/billing/common/renew-helper.service.js]: What do you think about setting `MMMM yyyy` manually?

File: client/app/account/billing/common/renew-helper.service.js [hunk 3]
```diff
@@ -1,65 +1,74 @@
...
             }
-            return $translate.instant(`autorenew_service_renew_${service}`);
+            return this.$translate.instant(translationId, { t0: this.$filter("date")(service.expiration, "mediumDate") });
         }
 
-        if (angular.isObject(service)) {
-            if (!_.isEmpty(service.subProducts)) {
-                return "";
+        getRenewLabel (service) {
+            if (service === "0") {
```
Comment (jleveugle) [inline client/app/account/billing/common/renew-helper.service.js]: Strange to test a number in string format :(

File: client/app/account/billing/common/renew-helper.service.js [hunk 4]
```diff
@@ -1,65 +1,74 @@
...
-        } else if (service.status === "EXPIRED") {
-            txt = "manuel";
-        } else if (this.serviceHasAutomaticRenew(service)) {
-            txt = "auto";
-            if (service.renew.period) {
-                txt = `frequency_value_${service.renew.period}`;
-            }
-        } else {
-            txt = "manuel";
+        /* eslint-disable class-methods-use-this */
```
Comment (jleveugle) [inline client/app/account/billing/common/renew-helper.service.js]: Disable ESLint is a **bad** idea :)

File: client/app/account/billing/components/renewDate/billing-renew-date.component.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (jleveugle) [inline client/app/account/billing/components/renewDate/billing-renew-date.component.js]: Do we have a specification which recommend this syntax ?

File: client/app/app.js (hunk 1 above)
Comment (Jisay) [reply client/app/app.js]: Are you sure that won't break nothing?

File: client/app/account/billing/common/renew-helper.service.js (hunk 3 above)
Comment (FredericEspiau) [reply client/app/account/billing/common/renew-helper.service.js]: Didn't refactor this

File: client/app/account/billing/common/renew-helper.service.js (hunk 4 above)
Comment (FredericEspiau) [reply client/app/account/billing/common/renew-helper.service.js]: Didn't refactor this 😄

File: client/app/account/billing/components/renewDate/billing-renew-date.component.js (hunk 5 above)
Comment (FredericEspiau) [reply client/app/account/billing/components/renewDate/billing-renew-date.component.js]: https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md

File: client/app/app.js (hunk 1 above)
Comment (FredericEspiau) [reply client/app/app.js]: I don't know:
* it's the value recommended by the documentation
* `"sanitizeParameters"` already breaks some characters

File: client/app/account/billing/common/renew-helper.service.js (hunk 2 above)
Comment (FredericEspiau) [reply client/app/account/billing/common/renew-helper.service.js]: Alright, alright, I'll refactor this class 😞

**Comment by jleveugle** [comment]
Hello @FredericEspiau,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**